meson.yaml

```diff
@@ -7,17 +7,18 @@ package:
     - license: Apache-2.0
   dependencies:
     runtime:
-      - python3
+      - python-3.11
       - samurai
 
 environment:
   contents:
     packages:
       - busybox
       - ca-certificates-bundle
-      - py3-setuptools
-      - python3
+      - py3.11-setuptools
       - samurai
```

Comment:
Is this not the standard way we bootstrap?

Reply:
I think this is perhaps a little suboptimal and we might want to look at making the solver in wolfictl a little more intelligent to deal with this edge case, but seems fine for now.